src/screens/FeedScreen/Content.js

```diff
@@ -233,21 +234,24 @@ const Content = ({
 
   const handleContainerText = () => {
     if (!showSeeMore && item.post_type === POST_TYPE_STANDARD && images_url.length <= 0) {
+      if (setIsShortText && typeof setIsShortText === 'function') {
+        setIsShortText(true);
+      }
       return {
         container: styles.centerVertical(item?.bg),
         text: styles.centerVerticalText(item?.color),
         isShort: true
       };
     }
+    if (setIsShortText && typeof setIsShortText === 'function') {
+      setIsShortText(false);
+    }
     return {
       container: styles.mv5,
       text: {},
       isShort: false
     };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `setIsShortText` is being called inside `handleContainerText`. Ensure that this function is always provided as a prop when the component is used, otherwise it will throw an error. Also, make sure that the function handles both true and false cases appropriately.
```diff
+ if (setIsShortText && typeof setIsShortText === 'function') {
+   setIsShortText(true);
+ }
...
+ if (setIsShortText && typeof setIsShortText === 'function') {
+   setIsShortText(false);
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->